internal/utils/config.go

```diff
@@ -285,6 +285,7 @@ type (
 	sms struct {
 		EnableSignup        bool              `toml:"enable_signup"`
 		EnableConfirmations bool              `toml:"enable_confirmations"`
+		Template            string            `toml:"template"`
```

Comment:
We should ensure backwards compatibility with existing config.toml which doesn't define this field. One way is to add a default value after line 122.

Reply:
done @sweatybridge